src/SdkCommon/ClientRuntime/Tests/TestCommon/Fakes/FakeHttpHandler.cs

```diff
@@ -27,10 +28,18 @@ public FakeHttpHandler()
         protected override Task<HttpResponseMessage> SendAsync(HttpRequestMessage request,
             CancellationToken cancellationToken)
         {
-            var response = new HttpResponseMessage(HttpStatusCode.OK);
+            var response = new HttpResponseMessage(HttpStatusCode.OK)
+            {
+                Content = new StringContent(String.Empty)
```

Comment:
We should make sure we still have tests with null Content : the above code handles it, but this could regress in the future.